bigbluebutton-html5/imports/ui/components/video-provider/many-users-notify/container.jsx

```diff
@@ -13,16 +14,21 @@ export default withTracker(() => {
   const meeting = Meetings.findOne({
     meetingId: Auth.meetingID,
   }, { fields: { 'usersProp.webcamsOnlyForModerator': 1, lockSettingsProps: 1 } });
+  const videoStreams = VideoStreams.find({ meetingId: Auth.meetingID },
+    { fields: { userId: 1 } }).fetch();
+  const videoUsersIds = videoStreams.map(u => u.userId);
   return {
     viewersInWebcam: Users.find({
       meetingId: Auth.meetingID,
-      hasStream: true,
+      userId: {
+        $in: videoUsersIds,
+      },
       role: ROLE_VIEWER,
       presenter: false,
     }, { fields: {} }).count(),
     currentUserIsModerator: Users.findOne({ userId: Auth.userID },
       { fields: { role: 1 } }).role === ROLE_MODERATOR,
-    lockSettings: meeting.lockSettingsProps,
+    lockSettings: Meetings.findOne({ meetingId: Auth.meetingID }).lockSettingsProps,
```

Comment:
Why do we need to look up Meetings again? Is this change needed? `meeting` should be identical